dotCMS/src/integration-test/java/com/dotcms/datagen/TemplateDataGen.java

```diff
@@ -47,6 +47,7 @@
 
     private static final TemplateAPI templateAPI = APILocator.getTemplateAPI();
     private static final String type = "template";
+    private boolean setBodyAsNull = false;
```

Comment:
Codacy found an issue: [Avoid using redundant field initializer for 'setBodyAsNull'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)